sympy/logic/inference.py

```diff
@@ -77,12 +78,24 @@ def satisfiable(expr, algorithm="dpll2", all_models=False):
     UNSAT
 
     """
+    if algorithm is None or algorithm == "pycosat":
+        pycosat = import_module('pycosat')
+        if pycosat is not None:
+            algorithm = "pycosat"
+        else:
+            # Silently fall back to dpll2 if pycosat
+            # is not installed
```

Comment:
If the algorithm is given explicitly as "pycosat" it should raise an exception in this case.